code/modules/projectiles/ammo_datums.dm

```diff
@@ -1010,6 +1010,19 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	shrapnel_chance = 25
 	sundering = 2.5
 
+/datum/ammo/bullet/railgun
+	name = "railgun round"
```

Comment:
From #4457
It can either be a railgun round or a high velocity projectile, up to you if you want to rename.